src/styles/container.scss

```diff
@@ -0,0 +1,9 @@
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin: 50px auto 0;
+  padding: 0 120px;
+  box-sizing: content-box;
+  max-width: 1024px;
```

Comment:
If we use max-width, it is also good to write width: 100%. Without width: 100% the width of the container depends on the width of the elements in the container